package/network/services/dnsmasq/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dnsmasq
-PKG_UPSTREAM_VERSION:=2.89
+PKG_UPSTREAM_VERSION:=2.90
 PKG_VERSION:=$(subst test,~~test,$(subst rc,~rc,$(PKG_UPSTREAM_VERSION)))
 PKG_RELEASE:=4
```

Comment:
`PKG_RELEASE` should also be reset to `1`.

Reply:
Please address your feedback to the original commit this is cherry-picked from.

Reply:
This is a cherry pick. Little or nothing should be changed from the base commit. Resolve this comment.

Reply:
Done.